From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bacteriawa <A3167717663@hotmail.com>
Date: Sun, 6 Jul 2025 03:03:12 +0800
Subject: [PATCH] Rebrand to LightingLuminol

diff --git a/src/main/java/io/papermc/paper/ServerBuildInfo.java b/src/main/java/io/papermc/paper/ServerBuildInfo.java
--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -18,6 +18,10 @@ public interface ServerBuildInfo {
      * The brand id for Paper.
      */
     Key BRAND_PAPER_ID = Key.key("papermc", "paper");
+    /**
+     * The brand id for LightingLuminol.
+     */
+    Key BRAND_LIGHTINGLUMINOL_ID = Key.key("luminolmc", "lightingluminol");
 
     /**
      * Gets the {@code ServerBuildInfo}.
