From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bacteriawa <A3167717663@hotmail.com>
Date: Mon, 29 Sep 2025 23:27:43 +0800
Subject: [PATCH] Rebrand to LightingLuminol


diff --git a/src/main/java/me/earthme/luminol/config/modules/misc/ServerModNameConfig.java b/src/main/java/me/earthme/luminol/config/modules/misc/ServerModNameConfig.java
index df5d926fb4401dd93d658fe14ae4d9d58cff68f9..86ad8d6e76ce361e806bfda97eb8a07cae1cfa08 100644
--- a/src/main/java/me/earthme/luminol/config/modules/misc/ServerModNameConfig.java
+++ b/src/main/java/me/earthme/luminol/config/modules/misc/ServerModNameConfig.java
@@ -8,7 +8,7 @@ import me.earthme.luminol.enums.EnumConfigCategory;
 @ConfigClassInfo(category = EnumConfigCategory.MISC, name = "server_mod_name")
 public class ServerModNameConfig implements IConfigModule {
     @ConfigInfo(name = "name")
-    public static String serverModName = "Luminol";
+    public static String serverModName = "LightingLuminol";
 
     @ConfigInfo(name = "vanilla_spoof")
     public static boolean fakeVanilla = false;
